crates/ruff_linter/src/codes.rs

```diff
@@ -972,6 +972,7 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         (Ruff, "036") => (RuleGroup::Preview, rules::ruff::rules::NoneNotAtEndOfUnion),
         (Ruff, "038") => (RuleGroup::Preview, rules::ruff::rules::RedundantBoolLiteral),
         (Ruff, "048") => (RuleGroup::Preview, rules::ruff::rules::MapIntVersionParsing),
+        (Ruff, "051") => (RuleGroup::Preview, rules::ruff::rules::UnrawRePattern),
```

Comment:
What's the motivation for picking 051 over 039 or 049?

Reply:
I'm happy to merge this soon. That's why I suggest we use `RUF039` for it, to avoid "holes". First come, first serve